externals/kyuubi-jdbc-engine/src/test/resources/doris-compose.yml

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+version: "3"
+services:
+  doris-fe:
```

Comment:
no official repo？

Reply:
yes，we can replace it after official repo release.

Reply:
Use https://hub.docker.com/r/apachedoris/doris-dev

Or 

Just use mysql directly, I heard but not quite sure myself, that doris is compatible with mysql

Reply:
the repo is a env of building code for doris developer, can't be used as a runtime.

The doris don't compatible with mysql, mysql driver is only used to talk with doris fe.

Reply:
it can't work well. I think that it's unnecessary to replace using one by a personal repo.

Reply:
<img width="1350" alt="image" src="https://user-images.githubusercontent.com/49054376/170263087-a842c468-4b32-4b59-a448-be62f6e2e78b.png">
it is unstable, was updated an hour ago.